logger/drain/tcp/drain_test.go

```diff
@@ -0,0 +1,13 @@
+package tcp
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestInvalidDrainUrl(t *testing.T) {
+	_, err := NewDrain("https://wwww.google.com")
+	if err == nil || err.Error() != fmt.Sprintf("Invalid drain url scheme: %s", "https") {
+		t.Error("Did not receive expected error message")
+	}
+}
```

Comment:
How about testing Send()? https://coderwall.com/p/wohavg/creating-a-simple-tcp-server-in-go